en/04_Changelogs/5.0.0.md

```diff
@@ -125,90 +190,90 @@ This is a major release and as a result there are a number of breaking API chang
-- `MailTransport`, which used the php native `mail()` function, is no longer officially supported in CMS 5. This is because [Symfony considers `mail()` to be insecure](https://github.com/swiftmailer/swiftmailer/issues/866#issuecomment-289291228).
-- If your site has a custom email configuration e.g. SMTP configuration, this will need to be updated, as the configuration has changed from Silverstripe yml to a much more flexible and standardised [DSN string](https://symfony.com/doc/current/mailer.html#transport-setup). See [the email documentation](/developer_guides/email/) for more details. The configuration for email has changed completely - please read the updated documentation carefully.
-- The following return types where changed on the following methods in [Email](api:SilverStripe\Control\Email\Email)
-  - Used to return a string, now returns an nullable [Address](https://symfony.com/doc/current/mailer.html#email-addresses) object
+- Email was sent in CMS 4 using [SwiftMailer](https://swiftmailer.symfony.com/docs/introduction.html), which has since [gone End Of Life](https://symfony.com/blog/the-end-of-swiftmailer). In CMS 5, this has been replaced with [symfony/mailer](https://symfony.com/doc/current/mailer.html). `symfony/mailer` is the currently maintained email package from Symfony. It's a more flexible email system that allows easier integration with third-party email providers.
+- In CMS 4, the [`Email`](api:SilverStripe\Control\Email\Email) class subclassed [`ViewableData`](api:SilverStripe\View\ViewableData). In CMS 5, it now subclasses `Symfony\Component\Mime\Email`.
+- `MailTransport`, which used the php native `mail()` function, is no longer present in CMS 5. This is because [Symfony considers `mail()` to be insecure](https://github.com/swiftmailer/swiftmailer/issues/866#issuecomment-289291228).
+- If your site has a custom email configuration e.g. SMTP configuration, this will need to be updated, as the configuration has changed from Silverstripe yml to a much more flexible and standardised [DSN string](https://symfony.com/doc/current/mailer.html#transport-setup). See [the email documentation](/developer_guides/email/) for more details. The configuration for email has changed completely - read the updated documentation carefully.
+- The following return types where changed on the following methods in [`Email`](api:SilverStripe\Control\Email\Email).
+  - Used to return a `string`, now returns an nullable [`Address`](https://symfony.com/doc/current/mailer.html#email-addresses) object.
     - `getReturnPath()`
     - `getSender()`
-  - Used to return an array of strings, now returns an array of `Address` objects
+  - Used to return an array of strings, now returns an array of `Address` objects.
     - `getFrom()`
     - `getTo()`
     - `getBCC()` - note - changed casing to `getBcc()`
     - `getCC()` - note - changed casing to `getCc()`
```

Comment:
Those methods are inherited from Symfony and don't show up in our API doc. So didn't add `api` links here.